### ISSUE TITLE
Qt and GTK can't be loaded on Linux

### ISSUE THREAD

**Issue by GabinTB** [r0x0r/pywebview#1211]
### Specification
- pywebview version: 4.3.2
- operating system: Linux
- web renderer: NA

### Description

When running the "get started" following code:
```Python
import webview
webview.create_window('Hello world', 'https://pywebview.flowrl.com/')
webview.start()
```
I get this error: `webview.util.WebViewException: You must have either QT or GTK with Python extensions installed in order to use pywebview.` while Qt AND GTK are properly installed.

**Comment by GabinTB** [comment]
Yes, everything is correctly installed and version checked

**Comment by gsal** [comment]
First time using pywebview.  Just installed a few minutes ago, also testing with the minimal example posted above.

The documentation says that
  - `pip install pywebview` will install it with the default dependencies...but it does not work.
  - `pip install pywebview[qt]` will install it with PySide2...but it dies not work. 

Tested in CentOS 7 and Rocky 9.

After installing with pywebview[qt], I get the following error: 

```python
python minimal.py 
Traceback (most recent call last):
  File "/home/salazag/programming/python/pywebview/minimal.py", line 4, in <module>
    webview.start(gui='qt')
  File "/home/salazag/.conda/envs/pyweb/lib/python3.11/site-packages/webview/__init__.py", line 155, in start
    guilib = initialize(gui)
             ^^^^^^^^^^^^^^^
  File "/home/salazag/.conda/envs/pyweb/lib/python3.11/site-packages/webview/guilib.py", line 108, in initialize
    if not try_import(guis):
           ^^^^^^^^^^^^^^^^
  File "/home/salazag/.conda/envs/pyweb/lib/python3.11/site-packages/webview/guilib.py", line 71, in try_import
    if import_func():
       ^^^^^^^^^^^^^
  File "/home/salazag/.conda/envs/pyweb/lib/python3.11/site-packages/webview/guilib.py", line 37, in import_qt
    import webview.platforms.qt as guilib
  File "/home/salazag/.conda/envs/pyweb/lib/python3.11/site-packages/webview/platforms/qt.py", line 350
    logger.warning('qtwebkit does not support _settings['private_mode']=False')
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
```

What gives?

**Comment by r0x0r** [comment]
@gsal This is a case of a botched find and replace refactor. I have fixed it now and the fix is released as 4.3.3

**Comment by r0x0r** [comment]
Regarding why GTK does not work on your system, I need more info.
I test with Ubuntu and it works here with the default install.

**Comment by gsal** [comment]
> @gsal This is a case of a botched find and replace refactor. I have fixed it now and the fix is released as 4.3.3

Nice, that was quick. 

I am up and running, now; thanks.

**Comment by GabinTB** [comment]
Already did this find and replace fiw but still can't run webview.
I tried the instruction here: https://github.com/r0x0r/pywebview/issues/805
```
sudo apt-get install build-essential libgl1-mesa-dev
pip3 install pyqt5 pyqtwebengine
pip3 install pywebview
pip3 install pywebview[qt]
```

And here: https://forum.qt.io/topic/127736/pyqt5-qtwebenginewidgets-importerror/9
` sudo apt-get install python3-pyqt5 python3-pyqt5.qtwebengine`, but neither worked.

What further information do you need to understand the "bug"?

**Comment by gsal** [comment]
It kind of looks like you have a different problem than mine; but, maybe you should try what I did. 

The last line in the minimal example, I did this:
`webview.start(gui='qt')`

Alternately, you can supposedly set an environment variable previous to running the program , to force the use of Qt:
 `export PYWEBVIEW_GUI=qt`

**Comment by gsal** [comment]
I am back.

I guess the hints above helped me get up and running; but, I ended up discovering that it is not working correctly, after all.

I have access to two platforms Centos7 and Rocky9.

Same installation process in each platform (python 3.9):
```bash
python -m venv venv
source ./venv/bin/activate
python -m pip install -U pip
pip install pywebview[qt]            # today, so, 4.3.3
``` 

Then:
 - I placed centos7 and rocky9 side-by-side
 - Tried 3 pywebview examples
 - On both systems, the sample apps come up and seem to do what it was intended: pop a dialog, hide the window, etc.
 - But
   - in centos7, the main window of the app shows content (advertising pywebview, user-agent, support)
   - in rocky9, the main window of the app remains blank; even when using "debug=True"
